### ISSUE TITLE
Image doesn't support the `alt` prop

### ISSUE THREAD

**Issue by Saadnajmi** [microsoft/react-native-windows#12228]
### Problem Description

https://github.com/facebook/react-native/commit/71fda5e0c2093380d08761c945f1e3029af6697f (0.71+) introduced the Image prop `alt`, which is a thin shim to `accessibilityLabel`. We are not properly supporting it in `Image.windows.js` or `Image.win32.js` it seems

Specifically, there is this logic in both `Image.android.js` and `Image.ios.js`:
>            accessibilityLabel={accessibilityLabel ?? props.alt}
that is not present in [Image.win32.js](https://github.com/microsoft/react-native-windows/blob/3571836656673331f936b43c4f1e1fa18b485871/packages/%40office-iss/react-native-win32/src/Libraries/Image/Image.win32.js#L216C35-L216C35) or [Image.windows.js](https://github.com/microsoft/react-native-windows/blob/c23fe1ab789c1d401b8a15edcb645d8a642ab7b3/vnext/src/Libraries/Image/Image.windows.js#L188C24-L188C24)

### Steps To Reproduce

N/A

### Expected Results

alt is shimmed in the JS layer

### CLI version

N/A

### Environment

```markdown
N/A
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by chrisglein** [comment]
Fixed with the integrate yesterday: https://github.com/microsoft/react-native-windows/commit/1c6a83c8bc10e5369791ac8f8b52dc04f8e1474a
So this will be fixed with 0.73. We could backport to 0.72 and 0.71. @Saadnajmi did we have a hit on a product that would indicate we should backport this? Should be a clear one to take.
Going to resolve this as fixed, but pop a comment if we need to backport.

**Comment by Saadnajmi** [comment]
> Fixed with the integrate yesterday: https://github.com/microsoft/react-native-windows/commit/1c6a83c8bc10e5369791ac8f8b52dc04f8e1474a
> 
> So this will be fixed with 0.73. We could backport to 0.72 and 0.71. @Saadnajmi did we have a hit on a product that would indicate we should backport this? Should be a clear one to take.
> 
> Going to resolve this as fixed, but pop a comment if we need to backport.

Yes, we hit it in product and I think backporting to 0.71 would be great! :)
